pkg/bootstrap/types.go

```diff
@@ -41,15 +40,24 @@ type OVNConfigBoostrapResult struct {
 	HyperShiftConfig *OVNHyperShiftBootstrapResult
 }
 
+type OVNUpdateStatus struct {
```

Comment:
Can you add some kind of comment about what this is for? Something about "the status of a Deployment or StatefulSet"?